src/Analysis/Codelyzer.Analysis.CSharp/Handlers/ClassDeclarationHandler.cs

```diff
@@ -21,20 +21,27 @@ public class ClassDeclarationHandler : UstNodeHandler
 
             if (classSymbol != null)
             {
-                if(classSymbol.BaseType != null)
+                ClassDeclaration.FullIdentifier = classSymbol.OriginalDefinition.ToString();
```

Comment:
@marknfawaz I think we should set the full identifier by calling `GetFullIdentifier()` function which returns namespace + "." + name.

`classSymbol.OriginalDefinition.ToString()` also returns namespace + "." + name with the type parameter which I don't think we should return, because it will change if the code changes. Take the `Modernize.Web.Models.Generics.ChildClass<ObjectType>` as an example. Currently, `classSymbol.OriginalDefinition.ToString()` will return "Modernize.Web.Models.Generics.ChildClass<ObjectType>". After I change the code to `Modernize.Web.Models.Generics.ChildClass<OT>`, it returns "Modernize.Web.Models.Generics.ChildClass<OT>". See the screenshot below.
<img width="689" alt="objectType" src="https://user-images.githubusercontent.com/108032883/178031153-843f83ec-0b09-487d-8def-e13aa84f55d7.png">


This may cause issues in the clients, since the full identifier changes if the code is changed. Let's have a sync up if it's not clear to you. Thanks.